sampler.go

```diff
@@ -456,6 +461,23 @@ func (s *PerOperationSampler) Close() {
 	s.defaultSampler.Close()
 }
 
+func (s PerOperationSampler) String() string {
+	var sb strings.Builder
+
+	fmt.Fprintf(&sb, "PerOperationSampler(defaultSampler=%v, ", s.defaultSampler)
+	fmt.Fprintf(&sb, "lowerBound=%f, ", s.lowerBound)
+	fmt.Fprintf(&sb, "maxOperations=%d, ", s.maxOperations)
+	fmt.Fprintf(&sb, "operationNameLateBinding=%t, ", s.operationNameLateBinding)
+	fmt.Fprintf(&sb, "currentOperationSize=%d,\n", len(s.samplers))
```

Comment:
```suggestion
	fmt.Fprintf(&sb, "numOperations=%d,\n", len(s.samplers))
```
similar to maxOperations

Reply:
Added